sdk/spring/azure-spring-boot-samples/azure-spring-boot-sample-active-directory-webapp/src/main/resources/application.yml

```diff
@@ -1,13 +1,14 @@
-# WebapiA is an optional client, we can access obo resource servers.
-# We can also access a custom server according to the webapiA client.
-
 azure:
   activedirectory:
     client-id: <client-id>
     client-secret: <client-secret>
     tenant-id: <tenant-id>
     user-group:
-      allowed-groups: group1, group2
+      allowed-groups: group1, group2    # By default, groups name is used for permission control.
```

Comment:
I have an idea, 
how about this
```
user-group:
  enable-group-id: true
  allowed-groups:
    - - groupName: groupName1
        groupId: groupId1 
      - groupName: groupName2
        groupId: groupId2
```
and I think by default groupId is optional, and groupName is necessary;
if user config groupName and groupId in pairs, we can check whether they are matching.

Reply:
@backwind1233, there's one defect of this design, which is the group name could is changeable. If we require the group name and when users change their group names, the application will stop working.

Reply:
@ZhuXiaoBing-cn has reverted the code

Reply:
it is wired to have both camel case and dash format within same properties file